terraform-common.mk

```diff
@@ -67,6 +67,10 @@ console: announce
 plan: announce .config $(TRVS_TFVARS) $(TFSTATE)
 	$(TERRAFORM) plan -module-depth=-1 -out=$(TFPLAN)
 
+.PHONY: pplan
+pplan: announce .config $(TRVS_TFVARS) $(TFSTATE)
+	$(TERRAFORM) plan -module-depth=-1 -out=$(TFPLAN) | ruby -ne 'puts $$_.gsub(/(metadata.user-data: *")(.*)(".*)/, "\\1blurp\\3")'
```

Comment:
FWIW, this is much the same reasoning behind the creation of the `plandiff` target, but I definitely understand how they differ.  I would prefer a slightly longer, more different than `plan` target name.  Maybe "planblurp" :sweat_smile: ?